src/Apps/Legion/Components/WizardShell/WizardShellMockData.ts

```diff
@@ -320,35 +320,35 @@ const setDairyData = (data: IWizardDataContextState) => {
     // TYPES
     const typePastTs = getType({
         friendlyName: 'Pasteurizer TS',
-        color: 'C13099',
+        color: '#C13099',
```

Comment:
Thank you for adding these, just please look at the `TypeIcon` component to make sure it didn't break, because I added the `#` in its styles file.

Reply:
Good callout. Fixed it.